modules/trino/src/test/java/org/testcontainers/containers/TrinoContainerTest.java

```diff
@@ -61,4 +61,13 @@ public void testInitScript() throws Exception {
             }
         }
     }
+
+    @Test
+    public void testExposedAndLivenessCheckPorts() {
+        try (TrinoContainer trino = new TrinoContainer(TrinoTestImages.TRINO_TEST_IMAGE)) {
+            trino.start();
+            assertThat(trino.getExposedPorts()).containsExactly(8080);
+            assertThat(trino.getLivenessCheckPortNumbers()).containsExactly(trino.getMappedPort(8080));
+        }
+    }
```

Comment:
I was thinking of merging a lot of these tests with the `testSimple` method. Some of these tests take a while and I'm not sure if it's worth it for the liveness check ports. So for this container
```
    @Test
    public void testSimple() throws Exception {
        try (TrinoContainer trino = new TrinoContainer(TrinoTestImages.TRINO_TEST_IMAGE)) {
            trino.start();
            try (
                Connection connection = trino.createConnection();
                Statement statement = connection.createStatement();
                ResultSet resultSet = statement.executeQuery("SELECT DISTINCT node_version FROM system.runtime.nodes")
            ) {
                assertThat(resultSet.next()).as("results").isTrue();
                assertThat(resultSet.getString("node_version"))
                    .as("Trino version")
                    .isEqualTo(TrinoContainer.DEFAULT_TAG);
                assertContainerHasCorrectPorts(trino);
            }
        }
    }
```

Reply:
I agree with this approach.

Reply:
- [ ] Merge tests into testSimple